content/blog/2019/03/2019-03-04-gsoc2019-announcement.adoc

```diff
@@ -0,0 +1,87 @@
+---
+:layout: post
+:title: "Jenkins is accepted to Google Summer Of Code 2019!"
+:tags:
+- gsoc
+- gsoc2019
+- events
+- community
+:author: oleg_nenashev
+:sig: gsoc
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+On behalf of the Jenkins GSoC org team,
+I am happy to announce that the Jenkins project **has been accepted** to
+link:https://summerofcode.withgoogle.com/[Google Summer of Code 2019].
+This year we invite students and mentors to join the Jenkins community and work together
+on enhancing the Jenkins ecosystem.
+
+Just to provide some numbers, this is the biggest GSoC ever, 206 organizations participate in GSoC this year.
+And it will be hopefully the biggest year for Jenkins as well.
+We have 25 link:/projects/gsoc/2019/project-ideas[project ideas]
+and more than 30 potential mentors (and counting!).
+It is already more than in 2016 and 2018 all together.
```

Comment:
Does this statement mean that 2019 numbers have exceeded that of 2016 and 2018 or does it mean that the 2019 numbers have exceeded 2016 and 2018 combined?

Reply:
2016 and 2018 combined

Reply:
I would note that is a combined number

Reply:
```suggestion
It is already more than in 2016 and 2018 combined.
```